e2e/e2e-script.sh

```diff
@@ -111,12 +135,14 @@ az vmss create -n ${VMSS_NAME} \
     --instance-count 1 \
     --assign-identity $msiResourceID \
     --image "microsoft-aks:aks:aks-ubuntu-1804-gen2-2021-q2:2021.05.19" \
-    --upgrade-policy-mode Automatic
+    --upgrade-policy-mode Automatic \
+    -ojson
```

Comment:
isn't -o json the default? just being explicit? not seeing the output used anywhere anyway?

Reply:
>isn't -o json the default? just being explicit?

It is, however we probably want the script to want on machines where a different output format is configured. On my laptop, for example, I had `table` configured, which made the script break.

>not seeing the output used anywhere anyway?

In this case the output isn't used, you're right. I set `-ojson` here, too, because the command's output is *printed* and I wanted to be consistent with other cases where we print stuff in JSON format.